src/field/FieldPropagator.hh

```diff
@@ -35,12 +35,11 @@ namespace celeritas
  *
  * \note This follows similar methods as in Geant4's G4PropagatorInField class.
  */
+template<class FieldT, template<class> class EquationT>
```

Comment:
Thinking about this more: the meta-template is not strictly *needed* here because the same class doesn't use different template instantiations of `EquationT`. The other place it's used (and indeed the only time I've **ever** had to use this construction) uses multiple templated class types of `T`: `T<A> a_; T<B> b_`.

Reply:
Will address this in a separate MR as suggested.